skribenten-web/frontend/src/routes/saksnummer_.$saksId.brevvelger.tsx

```diff
@@ -6,21 +6,15 @@ import { Outlet, useNavigate, useParams } from "@tanstack/react-router";
 import { groupBy, partition, sortBy } from "lodash";
 import { useState } from "react";
 
-import { getFavoritter, getLetterTemplate } from "~/api/skribenten-api-endpoints";
+import { getFavoritter } from "~/api/skribenten-api-endpoints";
 import { ApiError } from "~/components/ApiError";
 import type { LetterMetadata } from "~/types/apiTypes";
 
 export const Route = createFileRoute("/saksnummer/$saksId/brevvelger")({
   loaderDeps: ({ search: { vedtaksId } }) => ({ includeVedtak: !!vedtaksId }),
-  loader: async ({ context: { queryClient, getSakQueryOptions }, deps: { includeVedtak } }) => {
-    const sak = await queryClient.ensureQueryData(getSakQueryOptions);
-
-    const letterTemplates = await queryClient.ensureQueryData({
-      queryKey: getLetterTemplate.queryKey({ sakType: sak.sakType, includeVedtak }),
-      queryFn: () => getLetterTemplate.queryFn(sak.sakType, { includeVedtak }),
-    });
-
-    return { letterTemplates };
+  loader: async ({ context: { queryClient, getSakQueryOptions } }) => {
+    const sakContext = await queryClient.ensureQueryData(getSakQueryOptions);
```

Comment:
Dette fører ikke til to requests mot backend? Altså en gang når sakContext.sak hentes, og en gang når sakContext.brevMetadata hentes.

Reply:
Ah. Derfor bør vi bruke vedtaksId queryparam i parent routen egentlig, så slipper vi nokk det, så bør sakContext være ett parameter kanskje. Skal se litt på det.